### PR TITLE
smtk2ssrf-build.sh: Some changes in the script

### PR DESCRIPTION
**Pull request by salvadorcunat**
### Describe the pull request:

- [ ] Bug fix
- [X] Functional change
- [ ] New feature
- [ ] Documentation change
- [ ] Language translation

### Pull request long description:

Some scripting style changes and force rebuilding a stripped down version of subsurface to link against.
Please, see comments in individual commits.

### CODE REVIEW

File: scripts/smtk2ssrf-build.sh [hunk 1]
```diff
@@ -98,18 +110,55 @@ else
...
 fi
 
+cmake   -DBTSUPPORT=OFF \
+	-DCMAKE_BUILD_TYPE="$RELEASE" \
+	-DFBSUPPORT=OFF \
+	-DFORCE_LIBSSH=OFF \
+	-DFTDISUPPORT=OFF \
+	-DMAKE_TESTS=OFF \
+	-DNO_DOCS=ON \
+	-DNO_MARBLE=ON \
```
Comment (neolit123) [inline scripts/smtk2ssrf-build.sh]: we no longer have the `NO_MARBLE` option. Subsurface is always build with the Map widget (from QtLocation).

**Comment by mturkia** [comment]
It would probably be better to have a strict rule for proceeding

+[[ $_proceed == "n" ]] && exit 0

This will compile/continue with any typo, I would exit if not y, so any other response would be “safe”

**Comment by salvadorcunat** [comment]
Just sent patch fixing comments by @neolit123 and @mturkia

**Comment by neolit123** [comment]
thanks, LGTM.
@mturkia please merge if see fit.

**Comment by salvadorcunat** [comment]
Sorry.   Forgot to add this patch which just add a flag to build commandline mode without having to set it manually with cmake or ccmake.  This can be useful to @atdotde .
BTW I'm working a patches series working around the introduced commandline mode (get logs, warnings and so) but will send in another branch.

**Comment by neolit123** [comment]
we can merge this and amend later.
thanks @salvadorcunat !

**Pull request merged**